gapic-generator-java/src/main/java/com/google/api/generator/gapic/protoparser/HttpRuleParser.java

```diff
@@ -115,6 +115,10 @@ private static HttpBindings parseHttpRuleHelper(
             httpRule.getAdditionalBindingsList().stream()
                 .map(HttpRuleParser::getHttpVerbPattern)
                 .collect(Collectors.toList()))
+        .setAdditionalBindings(
+            httpRule.getAdditionalBindingsList().stream()
+                .map(x -> parseHttpRuleHelper(x, inputMessageOpt, messageTypes))
+                .collect(Collectors.toList()))
```

Comment:
`setAdditionalPatterns` above just gets the HttpVerb associated with the `additional_bindings` list. `setAdditionalBindings` recursively calls this function to get the nested `additional_bindings`.